internal/action/render.go

```diff
@@ -272,7 +271,7 @@ func populateDBRelatedImages(ctx context.Context, cfg *declcfg.DeclarativeConfig
 			}
 		}
 		for ri := range ris {
-			cfg.Bundles[i].RelatedImages = append(cfg.Bundles[i].RelatedImages, declcfg.RelatedImage{Image: ri})
+			cfg.Bundles[i].RelatedImages = append(cfg.Bundles[i].RelatedImages, declcfg.RelatedImage{Name: b.Name, Image: ri})
```

Comment:
This technically will change the ordering of related image contents in the rendered version of the index image as the underlying type is a set representation, and it looks like that ordering is based on the name key when playing around with this locally.